design-at-cornell/src/components/HomepageStyles.ts

```diff
@@ -131,4 +222,87 @@ export const Grid = styled.div`
   height: fit-content;
   margin-top: 90px;
   justify-content: center;
+  @media (max-width: 1000px) {
+    display: flex;
+    flex-direction: row;
+    height: fit-content;
+    justify-content: left;
+    margin-top: 0px;
+  }
+`;
+
+export const MobileElementContainer = styled.div`
```

Comment:
For MobileElementContainer, since it reuses a lot of the same styles as ElementContainer you could replace `export const MobileElementContainer = styled.div` with `export const MobileElementContainer = styled(ElementContainer)`. This will automatically contain all the same styles as ElementContainer and then you just need to add/change styles for whats different between ElementContainer and MobileElementContainer. In this case, I think you just need to change the margins since everything else is the same, so all you would need to include in MobileElementContainer is `margin-bottom: 90px; margin-inline: 20px;`